Packs/Automox/Playbooks/Upload_Vulnerabilities_to_Automox.yml

```diff
@@ -0,0 +1,362 @@
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": 760
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: b91e3b78-afd2-41cc-81ec-bc8a6f068957
+    type: regular
+    task:
+      id: b91e3b78-afd2-41cc-81ec-bc8a6f068957
+      version: -1
+      name: Approve Batch
+      description: Perform an action on an Automox Vulnerability Sync batch.
+      script: '|||automox-vulnerability-sync-batch-action'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      action:
+        simple: accept
+      batch_id:
+        simple: ${Automox.VulnUpload.batch_id}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": 920
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: fb737e18-cfa3-45be-8f60-dbe5d71d93fa
+    type: regular
+    task:
+      id: fb737e18-cfa3-45be-8f60-dbe5d71d93fa
+      version: -1
+      name: Get Report
+      description: Get details about a Vulnerability Sync batch.
+      script: '|||automox-vulnerability-sync-batch-get'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      batch_id:
+        simple: ${Automox.VulnUpload.batch_id}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": 1080
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: c75fd028-4c6a-493c-80fe-949448af5554
+    type: title
+    task:
+      id: c75fd028-4c6a-493c-80fe-949448af5554
+      version: -1
+      name: Done
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": 1270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1345,
+        "width": 630,
+        "x": 410,
+        "y": -10
+      }
+    }
+  }
+inputs:
+- key: entry_id
+  value: {}
+  required: true
+  description: The entryId of a vendor-generated vulnerability report.
```

Comment:
```suggestion
  description: The entryId of a vulnerability report.
```

Could we add something about how it must conform to one of the support report formats: https://support.automox.com/help/exporting-vulnerability-scanner-reports